art/estimators/classification/pytorch.py

```diff
@@ -401,7 +401,7 @@ def loss_gradient(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
 
         # Compute the gradient and return
         model_outputs = self._model(inputs_t)
-        loss = self._loss(model_outputs[-1], labels_t)
+        loss = self._loss(model_outputs[-1], labels_t.long())
```

Comment:
Why is this change required?

Reply:
Our modification is neglectable. This was just a temporary measure. I pushed the corrected version (i.e., the original version in IBM:dev_1.4.0).